app/views/research_sessions/preview.html.erb

```diff
@@ -114,6 +114,23 @@
       collected as part of the research is stored securely and kept strictly
       confidential.</p>
   </section>
+  <section>
+    <h3 class="subtitle-small" id="private">Concerns and complaints</h3>
```

Comment:
Is there a reason for the ID?

Reply:
We meant to link to individual sections, which we aren't currently doing, but to keep with the convention I've updated it to `concerns`